src/types.ts

```diff
@@ -284,3 +288,8 @@ export const clickhouseMigrationsDir = 'clickhouse/migrations';
 
 export const clickhouseMigrationFilenameMatch =
   /^(\d+)_([a-zA-Z_]+)\.(up|down)\.sql$/i;
+
+export type ServiceClient<T extends ServiceType> = {
+  instance: Client<T>;
+  garmr: GarmrService;
+};
```

Comment:
Added this so in the future we can define defaults for service integrations, all services should have a protobuf client and garmr instance